gr-qtgui/grc/qtgui_check_box.block.yml

```diff
@@ -15,7 +15,7 @@ parameters:
     option_labels: [Float, Integer, String, Boolean, Any]
     option_attributes:
         conv: [float, int, str, bool, eval]
-        cpp_opts: [double,int,std::string,double,double]
+        cpp_opts: ["double,int,std::string,double,double"]
```

Comment:
With this change cpp code generation fails as the yaml list now contains only one element instead of five, but if you only quote std::string  cpp generation works for me.